gen/templates/models/table/10_rel_load.go.tpl

```diff
@@ -239,11 +239,19 @@ func (os {{$tAlias.UpSingular}}Slice) Load{{$relAlias}}(ctx context.Context, exe
 
 	{{if $rel.IsToMany -}}
 		for _, o := range os {
+			if o == nil {
+				continue
+			}
+
 			o.R.{{$relAlias}} = nil
 		}
 	{{- end}}
 
 	for _, o := range os {
+		if o == nil {
+			continue
+		}
+
```

Comment:
I'm not sure these are necessary. The `All()` method will definitely not return any nil values.

Reply:
these two loops are iterating over the receiver of the method (`os`), not the result from `All()`. I added these checks when I kept running into panics after adding the check in `09_rel_query.go.tpl`

Reply:
#492 explains it with an example:

`Content` has an optional `to-one` relationship to `Sources`
`SourceFiles` has a `to-one` relationship to `Sources`, which implies a `to-many` relationship from the `Sources` side

So suppose we have the following relationship chains:

`content1` -> `source1` -> `sourcefile1`
`content2` -> (nothing)

executing 

```
Contents.Query(
		Preload.Content.Source(
			SelectThenLoad.Source.SourceFiles(),
		),
	).All(context.Background(), db)
```

will panic because these checks I'm updating are trying to `SelectThenLoad` `SourceFiles` on a `SourceSlice` that contain nil elements. IMO it makes sense for this Slice to have nil elements as `Preload.Content.Source` executes a left join, leaving the possibility of some of the results to be nil

Reply:
Actually..... I don't think this should happen. Before doing the `ThenLoad`, the slice should be stripped of all nil values.

I'll have to look into this.